script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -101,7 +101,7 @@ He's right.
 [sha:c06b38ac82190ab811bce412e4443d90dfa6d667]{
 -- Page 101, Offset 19918.
 -- ここは遠野邸前の国道で、通るのは乗用車ぐらい。
-The only thing that pass through the highway in front of the Tohno residence are passenger cars.
+The only things that pass through the highway in front of the Tohno residence are passenger cars.
```

Comment:
I really hate the word highway for 国道. It conjurs images of US interstates. Japanese roads are just different in their structure i guess, but I wonder if it might be better to just say "road" or something. Looking at google images, the 国道s in Tokyo just look like wide roads with on ramps.
> The only things that pass by on the highway in front of the Tohno residence are passenger cars.